addons/web/static/src/model/relational_model/utils.js

```diff
@@ -325,11 +325,9 @@ export function getFieldDomain(record, fieldName, domain) {
 }
 
 export function getBasicEvalContext(config) {
-    const { allowed_company_ids } = config.context;
     return {
         context: config.context,
-        allowed_company_ids,
```

Comment:
Why `allowed_company_ids` is removed and not `current_company_id` ?

Reply:
A mistake rebasing.